src/main/kotlin/xyz/atrius/waystones/data/config/Config.kt

```diff
@@ -132,6 +133,9 @@ class Config(plugin: KotlinPlugin) {
     // Gold grants 33% of the max boost per block
     private fun goldBoost(): Int = maxBoost() / 3
 
+    // Copper grants 25% of the max boost per block
+    private fun copperBoost(): Int = maxBoost() / 4
```

Comment:
Is copper rarer than iron? I don't play 1.17 that much currently so I haven't had much experience with it, but I was under the impression it was more common. If that's the case, it might be better to make it less powerful than iron (perhaps 10-12%?)